ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/components/CheckDetailsInfo.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {
+    Bullseye,
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    List,
+    ListItem,
+    Spinner,
+} from '@patternfly/react-core';
+
+import {
+    ComplianceCheckResult,
+    ComplianceClusterCheckStatus,
+} from 'services/ComplianceResultsService';
+
+import './CheckDetailsInfo.css';
+
+type CheckDetailsInfoProps = {
+    checkDetails?: ComplianceCheckResult | ComplianceClusterCheckStatus;
+    isLoading: boolean;
+    error?: Error;
+};
+
+function CheckDetailsInfo({ checkDetails, isLoading, error }: CheckDetailsInfoProps) {
+    if (error) {
+        return <div>Error: {error.message}</div>;
+    }
+
+    if (isLoading && !checkDetails) {
+        return (
+            <Bullseye>
+                <Spinner />
+            </Bullseye>
+        );
+    }
+
+    if (checkDetails) {
+        const { annotations, description, instructions, labels, rationale, warnings } =
+            checkDetails;
+        return (
+            <DescriptionList isHorizontal>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Description</DescriptionListTerm>
+                    <DescriptionListDescription className="formatted-text">
+                        {description}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Rationale</DescriptionListTerm>
+                    <DescriptionListDescription className="formatted-text">
+                        {rationale}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Instructions</DescriptionListTerm>
+                    <DescriptionListDescription className="formatted-text">
+                        {instructions}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Labels</DescriptionListTerm>
+                    <DescriptionListDescription className="formatted-text">
+                        <List isPlain>
+                            {Object.entries(labels).map(([key, value]) => (
+                                <ListItem key={key}>
+                                    {key}: {value}
```

Comment:
A question, not necessarily a suggestion, whether there is any convention in Kubernetes or OpenShift to render key in bold font weight?

Reply:
![CheckDetailsInfo](https://github.com/stackrox/stackrox/assets/11862657/65358b6f-c81a-4901-a4ba-43ce6b97abb1)

Reply:
From the few examples I see in OpenShift, they don't seem to bold the key. However, there aren't really examples of labels being listed out like this. Every where that I see a label, it's in... well... a `Label`. Maybe that's a better solution.
![Screenshot 2024-06-11 at 1 57 21 PM](https://github.com/stackrox/stackrox/assets/61400697/55b95151-34d0-4581-91b3-93a619db8c68)